front/src/config.js

```diff
@@ -1,7 +1,9 @@
 const apiUrls = {
-  develop: "http://localhost:3001/api/book?page=",
+  developFetch: "http://localhost:3001/api/book?page=",
```

Comment:
pisalem to juz wczesniej w jakims komentarzu. Tutaj tylko trzymaj apiUrl. wszystko inne trzymasz w api w odpowiadajacych funkcjach, tj. np. w fetchBooks podajesz url=`${apiUrl}/book?page=${pageNumber}`
Nie ma sensu trzymanie tego w roznych miejscach, jezeli chcesz szybko zobaczyc na jaki endpoint uderza dana metoda api

const apiUrls = {
develop: 'http://localhost:3001/api'
}

export const apiUrl = apiUrls[process.env.NODE_ENV]

i importujesz apiUrl w plikach z api

Reply:
Done.